### PR TITLE
Fixes to Argentina

### PR DESCRIPTION
**Pull request by waltergammarota**
Language fixes.
Added url to "what is my postal code"
Zip codes for downtown
Url in README corrected

### CODE REVIEW

File: app/locales/es-AR.json [hunk 1]
```diff
@@ -148,7 +148,11 @@
   "If you use this computer with multiple users, please use this link to update your status.": "Si utiliza este equipo con varios usuarios, por favor utilice este enlace para actualizar su estado.",
   "Write it down somewhere or save it on your computer.": "Anótela en algún lugar o guardelo en su computador.",
   "The link has been copied.": "El enlace se ha copiado.",
+<<<<<<< HEAD
```
Comment (michaelmcmillan) [inline app/locales/es-AR.json]: This needs to get removed :) It's basically git way of saying merge conflict. Just delete the line :)

File: app/locales/es-AR.json [hunk 2]
```diff
@@ -148,7 +148,11 @@
...
+  "clear this form": "borrar este formulario",
+=======
```
Comment (michaelmcmillan) [inline app/locales/es-AR.json]: Same here

File: app/locales/es-AR.json [hunk 3]
```diff
@@ -148,7 +148,11 @@
...
   "clear this form": "borrar esta forma",
+>>>>>>> upstream/master
```
Comment (michaelmcmillan) [inline app/locales/es-AR.json]: and here :)

**Review by michaelmcmillan — CHANGES_REQUESTED** [review]
merge conflict lines in the code :) good if you remove those!

**Comment by michaelmcmillan** [comment]
See comments above @waltergammarota :)

File: README.md [hunk 4]
```diff
@@ -25,8 +25,12 @@ We don't know how many people have COVID-19. So we made a website where people c
 - 🇨🇦 Canada: https://coronastatus.ca
 - 🇦🇺 Australia: https://coronastatusau.org
 - 🇸🇬 Singapore: https://coronastatus.sg
+- 🇦🇷 Argentina: https://coronastatus.com.ar
 - 🇲🇾 Malaysia: https://coronastatusmy.org
+<<<<<<< HEAD
```
Comment (fossecode) [inline README.md]: Little merge conflict here that needs to be resolved :)

Comment (waltergammarota) [reply README.md]: Yes I just saw it. Do you know any trick to catch them before doing a push?

Comment (fossecode) [reply README.md]: When you try to merge master into your branch, you will be informed that you have merge conflicts. If you then run `git status`, you will see which files that have conflicts. These need to be resolved before running `git commit`.

**Review by fossecode — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (waltergammarota) [reply README.md]: Thanks!

**Pull request merged**